### PR TITLE
feat(docs): Add to "Non-Consensus Breaking Release Process"

### PR DESCRIPTION
**Pull request by gregorycoppola**
## Description

We add some new steps to the release process documentation, after having been release manager.

## Type of Change
* New feature

## Testing information
No testing, just docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,7 +329,24 @@ cargo fmt --all
 
 For non-consensus breaking releases, this project uses the following release process:
 
-1. A release manager enumerates the PRs or issues that would _block_
+1. The release must be timed so that it does not interfere with a *prepare
+phase*.  The timing of the next Stacking cycle can be found
+[here](https://stacking.club/cycles/next). A release to `mainnet` should happen
+at least 24 hours before the start of a new cycle, to avoid interfering
```
Comment (gregorycoppola) [inline README.md:335]: Does anyone want to add anything to determining how to avoid the *prepare phase*? @kantai ?

Comment (kantai) [reply README.md:335]: This looks good to me. Another possible source is https://blockstack.github.io/when-rewards/

**Review by kantai — APPROVED** [review]
LGTM! Thanks for adding @gregorycoppola

**Review by pavitthrap — APPROVED** [review]
LGTM, thanks for improving the docs!

**Pull request merged**